### PR TITLE
[NEW] Add Leader functionality to groups

### PR DESCRIPTION
**Pull request by danischreiber**
[NEW] Add a leader role so that each group can have one user set up to be displayed at the top of the room UI, with a direct chat link. This is being done after discussion with the RocketChat folks, for SeekingAlpha integration.
Please take a look ASAP.

### CODE REVIEW

**Comment by k0nsl** [comment]
Why deviate from the term 'moderator' to 'leader' for groups? For channels we use the term 'moderator' so why should this be different for groups?

**Comment by danischreiber** [comment]
A moderator can control the group in a way that a leader does not. The
leader is merely an important person in the discussion. This was the
conclusion we reached in discussion with Rodrigo, Sing Li, etc.

**Comment by k0nsl** [comment]
Thank you for the clarification, @danischreiber. That makes sense, indeed.

File: packages/rocketchat-authorization/server/startup.js [hunk 1]
```diff
@@ -73,6 +74,7 @@ Meteor.startup(function() {
 	const defaultRoles = [
 		{ name: 'admin',     scope: 'Users',         description: 'Admin' },
 		{ name: 'moderator', scope: 'Subscriptions', description: 'Moderator' },
+		{ name: 'leader', 	 scope: 'Subscriptions', description: 'Leader' },
```
Comment (rodrigok) [inline packages/rocketchat-authorization/server/startup.js]: Can you fix the spaces here to keep the indentation? Seems that you are using tabs

Comment (danischreiber) [inline packages/rocketchat-authorization/server/startup.js]: Fixed in new push

**Comment by danischreiber** [comment]
Opening a new pull request from a separate feature branch to make this cleaner.

**Pull request closed without merging**